Snakefile

```diff
@@ -55,47 +64,60 @@ rule all:
         ],
 
 
-def devtools_install():
-    if config["ploidetect_local_clone"] and config["ploidetect_local_clone"] != "None":
-        install_path = config["ploidetect_local_clone"].format(**config)
-        devtools_cmd = f"devtools::install_local('{install_path}', force = TRUE)"
-    else:
-        ver = config["ploidetect_ver"]
-        devtools_cmd = f"devtools::install_github('lculibrk/Ploidetect', ref = '{ver}')"
-    return f'"{devtools_cmd}"'
-
 rule download_cytobands:
     """Downloads cytoband data for plotting & (todo) hgver-specific centromere filtering"""
     input:
         HTTP.remote(
-            expand("http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz", hgver = config["genome_name"])
-        )
+            expand(
+                "http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz",
+                hgver=config["genome_name"],
+            )
+        ),
     output:
-        expand("resources/{hgver}/cytobands.txt", hgver = config["genome_name"])
+        expand("resources/{hgver}/cytobands.txt", hgver=config["genome_name"]),
```

Comment:
@lculibrk - I think a number of these changes are just from snakefmt.  It code prettifies - built into the make --lint